cmd/admin/email.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+
+	"github.com/letsencrypt/boulder/sa"
+)
+
+// subcommandClearEmail encapsulates the "admin clear-email" command.
+//
+// Note that this command may be very slow, as the initial query to find the
+// set of accounts which have a matching contact email address does not use a
+// database index. Therefore, when removing contact email addresses from found
+// accounts, it does not exit on failure, preferring to continue and make as
+// much progress as possible.
+func (a *admin) subcommandClearEmail(ctx context.Context, args []string) error {
+	subflags := flag.NewFlagSet("clear-email", flag.ExitOnError)
```

Comment:
I think this should be a command like "update-email", with a -clear flag.

That provides an obvious place to add an update option in the future.

I think it would be best to have a flag with a list of registration IDs to update, and a separate lookup command to run ahead of time.  That way the actual clear is fast (since we can just touch those rows), and can code-review the implementation plan to remove a user's contact information and be sure which registrations it'll touch.

We probably want the registrations flag to be optional in case we need to remove in bulk (eg, if somebody asks to remove an email from hundreds or thousands of accounts, perhaps because they register each time)

Reply:
I think these are great ideas for a follow-up change. Right now this change does not add any new functionality, and does not meaningfully change any of the existing functionality, with the exception of unifying which revocation commands can take which flags. I really want to keep this PR scoped to "replace admin-revoker", not "and add a bunch of new utilities". Can these ideas be tracked on https://github.com/letsencrypt/boulder/issues/6864 instead?

Reply:
I don't think I'm suggesting additional functions here; this is suggesting how to name and structure the new commands to allow for future expandability. 

I'm just saying you should name the command "update-email" with a mandatory -clear flag.

 I don't really care whether it's in this PR or a future one